index.html

```diff
@@ -944,14 +944,20 @@ <h3>
           Cross-origin Resources
         </h3>
         <p class="note" data-dfn-for="PerformanceResourceTiming">
-          As detailed in [=Fetch=], cross-origin resources are included as
-          <a>PerformanceResourceTiming</a> objects in the <a data-cite=
-          "PERFORMANCE-TIMELINE-2#performance-timeline">Performance
-          Timeline</a>. If the <a data-cite="FETCH#concept-tao-check">timing
-          allow check</a> algorithm fails for a resource, the following
-          attributes of its <a>PerformanceResourceTiming</a> object are set to
-          zero: {{PerformanceResourceTiming/redirectStart}},
+          As detailed in [=Fetch=], requests for cross-origin resources are
+          included as <a>PerformanceResourceTiming</a> objects in the
+          <a data-cite= "PERFORMANCE-TIMELINE-2#performance-timeline">
+          Performance Timeline</a>. If the <a
+          data-cite="FETCH#concept-tao-check">timing allow check</a> algorithm
+          fails for a cross-origin resource, the entry will be an
+          <a data-cite="FETCH#create-an-opaque-timing-info">opaque entry</a>.
```

Comment:
Nit: replace the data-cite with ` [=create an opaque timing info|opaque entry=]`